pom.xml

```diff
@@ -37,14 +37,32 @@
               <!-- stream benchmark under BSD-like license with a few exceptions around benchmarking results -->
               <exclude>tools/bin/src/stream/*</exclude> 
 
-              <!-- PyGreSQL an open-source Python module that interfaces to a PostgreSQL database under the Python license -->
+              <!-- exclude open-source python modules -->
+              <!-- MIT license for open-source python modules -->
               <exclude>tools/bin/pythonSrc/PyGreSQL-4.0/**</exclude>
+              <exclude>tools/bin/pythonSrc/PSI-0.3b2_gp/**</exclude>
+              <exclude>tools/bin/pythonSrc/lockfile-0.9.1/**</exclude>
+              <!-- BSD license for open-source python modules -->
+              <exclude>tools/bin/pythonSrc/pychecker-0.8.18/**</exclude>
+              <exclude>tools/bin/pythonSrc/unittest2-0.5.1/**</exclude>
```

Comment:
PKG-INFO is wrong (just noticed).

tools/bin/pythonSrc/unittest2-0.5.1]$ head setup.py

#! /usr/bin/env python
# setup.py
# Install script for unittest2
# Copyright (C) 2010 Michael Foord
# E-mail: fuzzyman AT voidspace DOT org DOT uk

# This software is licensed under the terms of the BSD license.
# http://www.voidspace.org.uk/python/license.shtml

import os

2016-07-13 7:58 GMT+08:00 Roman V Shaposhnik <notifications@github.com>:

> In pom.xml
> <https://github.com/apache/incubator-hawq/pull/788#discussion_r70544327>:
>
> >                <exclude>tools/bin/pythonSrc/PyGreSQL-4.0/**</exclude>
> > +              <exclude>tools/bin/pythonSrc/PSI-0.3b2_gp/**</exclude>
> > +              <exclude>tools/bin/pythonSrc/lockfile-0.9.1/**</exclude>
> > +              <!-- BSD license for open-source python modules -->
> > +              <exclude>tools/bin/pythonSrc/pychecker-0.8.18/**</exclude>
> > +              <exclude>tools/bin/pythonSrc/unittest2-0.5.1/**</exclude>
>
> unittest2-0.5.1/PKG-INFO says License is unknown. Please provide support
> for your claim that this is a BSD licensed code.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/incubator-hawq/pull/788/files/0c825258cc886917477b60f706a87e7d3dd831f4#r70544327>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHI5jGA59Fdr2gVGgdY5tZOVHFkHDYy_ks5qVCo7gaJpZM4JKQQE>
> .
>